src/renderer/Menu.tsx

```diff
@@ -52,6 +57,72 @@ export interface MenuProps {
 
 const Menu: React.FC<MenuProps> = function ({ t, error }: MenuProps) {
 	const classes = useStyles();
+
+	const [settings, setSettings] = useContext(SettingsContext);
+
+	const [launchItemList, setLaunchItemList] = useState([] as any[]);
+	const [openMessage, setOpenMessage] = useState(<>{t('game.error_platform')}</>);
+	const [dropdownOpen, setDropdownOpen] = useState(false);
+
+	const anchorRef = useRef(null);
+
+	const toggleDropdownOpen = () => { setDropdownOpen((status) => !status); };
+
+	// Grab available platforms from main thread
+	useEffect(() => {
+		ipcRenderer.invoke(IpcMessages.REQUEST_PLATFORMS_AVAILABLE, settings.launchPlatformSettings).then((result: GamePlatformMap) => {
+			setSettings({
```

Comment:
Why is it saved in a setting instead of a state? At the moment I don't see the point of saving the whole list when it is requested anyway at startup. But I might be wrong.

Reply:
You know, that's a good point. The renderer basically only needs to know about the key, availability, and translate entry. I'll refactor this so most stuff is handled only in the main thread. So I can cut it to state, AND cut down on the data stored in it.